docs/source/modules/cmci_action.rst

```diff
@@ -244,7 +244,7 @@ resources
 
      
   get_parameters
-    A list of one or more parameters with optional values used to identify the resources for this request. Eligible parameters for identifying the target resources can be found in the resource table reference for the target resource type, as valid parameters for the GET operation in the "Valid CPSM operations" table. For example, the valid parameters for identifying a PROGDEF resource are CICSSYS, CSDGROUP and RESGROUP, as found in the `PROGDEF resource table reference <https://www.ibm.com/support/knowledgecenter/en/SSGMCP_5.6.0/reference-cpsm-restables/cpsm-restables/PROGDEFtab.html>`_.
+    A list of one or more parameters with optional values used to  identify the resources for this request. Eligible parameters for identifying the target resources can be  found in the resource table reference for the target resource type, as valid parameters for the GET operation in  the "Valid CPSM operations" table. For example, the valid parameters  for identifying a PROGDEF resource are CICSSYS, CSDGROUP and  RESGROUP, as found in the `PROGDEF resource table reference <https://www.ibm.com/support/knowledgecenter/en/SSGMCP_5.6.0/reference-cpsm-restables/cpsm-restables/PROGDEFtab.html>`_.
```

Comment:
Looks like you have a bunch of extra spaces in here now?

Reply:
I guess this is because they're generated - any way we can avoid this?

Reply:
@stewartfrancis Interesting, at first I thought it's because an extra space at the end of the line, which seems to fix the issue in the generated rst, but there're other lines with extra spaces in the end that do not show this problem.

With extra space or not, I checked the html output, which look fine in either case, so shouldn't be an issue with the final output. 

However, I removed all the extra spaces, just in case.